resources/views/pages/settings/password.twig

```diff
@@ -0,0 +1,36 @@
+{% extends 'pages/settings/settings.twig' %}
+{% import 'macros/form.twig' as f %}
+{% import 'macros/base.twig' as m %}
+
+{% block title %}{{ __('settings.password') }}{% endblock %}
+
+{% block row_content %}
+        <form action="" enctype="multipart/form-data" method="post">
+            {{ csrf() }}
+
+            <div class="row">
+                <div class="col-md-12">
+                    {{ f.info(__('Here you can change your password.')) }}
+                    {{ f.input(
+                        'password',
+                        __('Old password:'),
+                        'password',
+                        {'required': true}
+                    ) }}
+                    {{ f.input(
+                        'new_password',
+                        __('New password:'),
+                        'password',
+                        {'required': true}
```

Comment:
I would definitely also add a html `minlength` attribute here (as `min` opt) which gets configured by the config which should mean that one sentence has to be translated less :upside_down_face:

Reply:
Added minlength to registration and password reset as well.